test/make.sh

```diff
@@ -132,15 +150,18 @@ function generate() {
 
```

Comment:
Do we still need [`check_generate`](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/223/files#diff-e9cfabf9b162cebf5922498a77b8f766R171-R191) and [`check_generate_docs`](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/223/files#diff-e9cfabf9b162cebf5922498a77b8f766R193-R212) below ? 

If you're going to update the helper functions, please keep make targets to be updated in mind ;)

Reply:
Didn't get your question, both function used at the moment to check terraform and docs files generation

Reply:
we just don't have them in the [latest version](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/test/make.sh) of the helper functions. I just wanted to make sure that they didn't get outdated yet